main.go

```diff
@@ -0,0 +1,31 @@
+/*
+ *Author: Stefan
+ *Date: 11/16/2019
+ *Task: Home work 2.1 Implement - func factorial(i uint) uint - function without using recursion
+ *Examples: Factorial(0) = 1; For enother: Factorial(3) = 1 * 2 * 3 = 6; Factorial(5) = 1 * 2 * 3 * 4 * 5 = 120;
+**/
+
+package main
+
+import "fmt"
+
+func factorial(i uint) uint {
+
+	if i == 0 {
+		return 1
+	}
+
+	var result uint = 1
+	var a uint = 2
```

Comment:
Can be simplified with
```go 
	for a := uint(2); a <= i; a++ {
		result *= a

	}
```

Reply:
I absolutely agree! But there were other options in the lecture. Thanks a lot, great!